google/cloud/video/transcoder_v1/types/resources.py

```diff
@@ -74,8 +81,10 @@ class Job(proto.Message):
             -  ``preset/{preset_id}``
 
             -  User defined JobTemplate: ``{job_template_id}``
+            This field is a member of `oneof`_ ``job_config``.
```

Comment:
We can apply a fix manually as this has been fixed in the generator
https://github.com/googleapis/gapic-generator-python/issues/1064.

```suggestion

            This field is a member of `oneof`_ ``job_config``.
```